tests/google-cloud-python/Makefile

```diff
@@ -0,0 +1,3 @@
+.PHONY: all
+all:
+	docker build --no-cache .
```

Comment:
Build error unrelated. We should probably turn off travis because the 3.5 interpreter build takes way too long for travis. Either that or find a way to cache the 3.5 build.